config/ApplicationUnitTest.cmake

```diff
@@ -243,27 +243,63 @@ macro( aut_register_test )
     message( FATAL_ERROR "Python not found in PATH")
   endif()
 
-  add_test(
-    NAME ${ctestname_base}${argname}
-    COMMAND ${PYTHON_COMMAND}
-    ${aut_DRIVER}
-    -DAPP=${aut_APP}
-    -DARGVALUE=${argvalue}
-    -DWORKDIR=${aut_WORKDIR}
-    -DTESTNAME=${ctestname_base}${argname}
-    -DDRACO_CONFIG_DIR=${DRACO_CONFIG_DIR}
-    -DDRACO_INFO=$<TARGET_FILE_DIR:Exe_draco_info>/$<TARGET_FILE_NAME:Exe_draco_info>
-    -DSTDINFILE=${aut_STDINFILE}
-    -DGOLDFILE=${aut_GOLDFILE}
-    -DRUN_CMD=${RUN_CMD}
-    -DnumPE=${numPE}
-    -D MPIEXEC=${MPIEXEC}
-    -DMPI_CORES_PER_CPU=${MPI_CORES_PER_CPU}
-    -DSITENAME=${SITENAME}
-    -DPROJECT_BINARY_DIR=${PROJECT_BINARY_DIR}
-    -DPROJECT_SOURCE_DIR=${PROJECT_SOURCE_DIR}
-    ${BUILDENV}
-    )
+  # Check to see if driver file is python or CMake
+  set(PYTHON_TEST 0)
+  string(FIND $aut_DRIVER ".py" PYTHON_TEST)
+  if ($PYTHON_TEST EQUAL -1)
+    set($PYTHON_TEST FALSE)
+  else ()
+    set($PYTHON_TEST TRUE)
+  endif ()
+
+
+  # Add python version if python driver file is specified
+
+  if ($PYTHON_TEST)
```

Comment:
* Does this syntax work in CMake now?  I though you always needed the curly braces: `${PYTHON_TEST}`.
* I'm pretty sure that you don't use the `$` when setting a variable (line 250, 252).